docs/cel2/operators/migrate-node.md

```diff
@@ -27,105 +28,140 @@ Migrating a pre-hardfork datadir involves these high-level steps:
 ### Important Notes
 
 - The migration tool can be run multiple times as the L1 chain data grows and will continue migrating from where it last left off.
-
+- While the pre-migration can be run multiple times and will get faster each time, you should avoid running the full migration more than once as it will be slower the second time.
 - All migrations writing to a given destination datadir must use the same node's source datadir. That is, you should not run the pre-migration with a db snapshot from node A and then run the full migration with a db snapshot from node B.
-
 - Your node must be stopped before the migration tool is run, even once it has reached the hardfork.
-
 - You should not attempt to migrate archive node data, only full node data.
 
-- While the pre-migration can be run multiple times and will get faster each time, you should avoid running the full migration more than once as it will be slower the second time.
-
-## Before the hardfork
+## Preparation Steps
 
-There are some important steps node operators should take ahead of the L2 hardfork. These include upgrading to the [latest client release](run-node.md#network-config--assets) so that the L1 network will stop producing blocks at the hardfork, and __running a pre-migration 1-2 days before the hardfork__. If you have not yet read the [Preparing for the L2 migration](../notices/l2-migration.md) page, please do so before continuing.
+### 1. Upgrade L1 Nodes
 
-## Running the migration
+All node operators must upgrade their L1 (`celo-blockchain`) nodes to the required version before the hardfork. This release defines migration block numbers so nodes will stop producing blocks at the right time.
 
-:::warning
-__It is not recommended to migrate from an L1 archive datadir.__
+### 2. Run a Pre-Migration (Recommended)
 
-If you only have an L1 archive node, we recommend syncing an L1 full node in preparation for the Mainnet migration. You can still run an L2 archive node after migrating from an L1 full node datadir. See [Running a Celo archive node](run-node.md#running-an-archive-node) for more.
+:::warning Archive Node Limitation
+Both pre-migration and full migration require **full node data only**. If you only have archive nodes, sync a full node before the hardfork. You cannot migrate archive data, even for L2 archive nodes. See [Running an archive node](/cel2/operators/run-node.md#running-an-archive-node) for details.
 :::
 
-:::warning
-__We have sometimes encountered the following problem when migrating an L1 datadir.__
+Run a pre-migration 1-2 days before the hardfork to migrate most data and minimize downtime.
```

Comment:
This can be removed, now that we're past the migration.

Reply:
removed